src/App.js

```diff
@@ -2,18 +2,68 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import { Form } from './api/components/Form';
+import { TodoList } from './api/components/TodoList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+export class App extends React.Component {
+  state ={
+    todoList: [],
+    title: '',
+    userName: '',
+    classValidTitle: '',
+    classValidUser: '',
+  }
```

Comment:
лінтер чому відключений?
```suggestion
  state = {
    todoList: [],
    title: '',
    userName: '',
    classValidTitle: '',
    classValidUser: '',
  }
```